### PR TITLE
Refactor release automation to accomodate for nectar

### PR DESCRIPTION
**Pull request by thomaseizinger**
A successful run of these can be seen here: https://github.com/thomaseizinger/comit-rs/releases

### CODE REVIEW

File: .github/workflows/draft-new-cnd-release.yml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+      - name: Update changelog
+        uses: thomaseizinger/keep-a-changelog-new-release@v1
+        with:
+          version: cnd-${{ github.event.inputs.version }}
+          changelogPath: cnd/CHANGELOG.md
+
+      - name: Initialize mandatory git config
+        run: |
+          git config user.name "COMIT Botty McBotface"
+          git config user.email botty@coblox.tech
```
Comment (D4nte) [inline .github/workflows/draft-new-cnd-release.yml:31]: Is that an actual address?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -55,7 +55,7 @@ Please be sure to double check your commit history and try to keep it clean, esp
 New code needs to be accompanied by **new tests**. Please use existing tests as example.
 We are using Continuous Integration and having tests will not only ensure that your code is correct but also that no-one else is breaking it unintentionally.
 
-Finally, do not forget to update the [changelog](./CHANGELOG.md).
+Finally, do not forget to update the [changelog](cnd/CHANGELOG.md).
```
Comment (D4nte) [inline CONTRIBUTING.md:58]: Should we just remove the link? Do we have a changelog for the comit lib?

**Review by D4nte — APPROVED** [review]
Well done.

File: .github/workflows/draft-new-cnd-release.yml (hunk 1 above)
Comment (thomaseizinger) [reply .github/workflows/draft-new-cnd-release.yml:31]: Yes :)

File: CONTRIBUTING.md (hunk 2 above)
Comment (thomaseizinger) [reply CONTRIBUTING.md:58]: We don't have one yet but once it is released I would add one :)

I actually didn't update this section manually, thanks for pointing it out!

File: .github/actions/create-release-archive/action.yml [hunk 3]
```diff
@@ -0,0 +1,29 @@
+name: Create release archive
+description: Creates a tar archive for a release binary
+inputs:
+  version:
+    description: 'The version of the binary'
+    required: true
+  binary:
+    description: 'The name of the binary to pack into the archive'
```
[the remark this answers is not in the document]
Comment (thomaseizinger) [reply .github/actions/create-release-archive/action.yml:8]: It also works for other Rust binaries if they exist :D

From the PoV of this action, I wouldn't bother :)

**Pull request merged**